### PR TITLE
change backend from bookstore to echo

### PR DESCRIPTION
**Pull request by wlu2016**
- Use echo samples for consistency with demos on gce/mvm.                                                                         |
- echo samples is already available in github.                                                                                    |
- bookstore samples are not ported to github yet.                                                                                 |
- Also add test cert and test key for https demo.

### CODE REVIEW

File: gke/nginx.key [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (jonparrott) [inline gke/nginx.key]: Should this be here? Private keys are almost never supposed to be in git.

**Comment by wlu2016** [comment]
this is a test key. Otherwise, we would ask user to generate it themselves. I am fine if policy doesn't allow any private key even for testing purpose.

**Comment by jonparrott** [comment]
Gotcha. That's fine, I suppose, but it's not really unusual to request the user to run an openssl command to generate those.

**Comment by wlu2016** [comment]
do you happen to know how to run this openssl command to generate the key/cert?

I am following this doc, which requires a certain setup effort (like install go, build binary): 
https://github.com/kubernetes/kubernetes/tree/master/examples/https-nginx

**Comment by elibixby** [comment]
I'd rather leave out the cert and key, and include the instructions for the key and cert instead. I think the reduction in steps isn't really worth the off chance that someone will forget to replace the cert and key in developing their own API, and then have someone able to spoof their backend.

**Comment by wlu2016** [comment]
done, ptal

**Comment by elibixby** [comment]
LGTM. As a follow up, it sounds like there was a decision to recommend docker on GCE? Do we want to update `gce/startup.sh` to use docker rather than directly running the python backend?

**Comment by wlu2016** [comment]
Dan briefly mentioned this to me, but I am not sure about if the decision has been made. 

Thanks for merging this change :) does that happened automatically after you lgtm? or do you need to click "merge" manually?

**Comment by elibixby** [comment]
I clicked merge manually.

**Pull request merged**